raspi-config

```diff
@@ -276,6 +276,9 @@ calc_wt_size() {
   # stderr is a tty and so only gives default 80, 24 values
   WT_HEIGHT=18
   WT_WIDTH=$(tput cols)
+  WT_BOX_HEIGHT=20
+  WT_BOX_WIDTH=60
+  WT_BOX_WIDTH_WIDE=70
```

Comment:
Is it actually necessary to have two different widths, given they differ by so little? Might be tidier to just have one?

Reply:
I'd suggest using whichever is least risky - presumably the wider one - and testing to make sure it is still ok with the other messages. I'd assume the narrower one was historically the default but at some point it was found to cause problems with one or two particular messages.